content/actions/creating-actions/creating-a-docker-container-action.md

```diff
@@ -96,7 +96,7 @@ runs:
 ```
 {% endraw %}
 
-This metadata defines one `who-to-greet`  input and one `time` output parameter. To pass inputs to the Docker container, you must declare the input using `inputs` and pass the input in the `args` keyword.
+This metadata defines one `who-to-greet`  input and one `time` output parameter. To pass inputs to the Docker container, you should declare the input using `inputs` and pass the input in the `args` keyword. Everything you pass to the `args` will be passed to the container, but for readability purposes, it is best practice to use inputs.
```

Comment:
@nikola-jokic Thanks a lot for this fix! I've suggested a slight rewording below; Instead of "but for readability purposes", would "but for better discoverability for users of your action" also be accurate?

```suggestion
This metadata defines one `who-to-greet`  input and one `time` output parameter. To pass inputs to the Docker container, you should declare the input using `inputs` and pass the input in the `args` keyword. Everything you include in `args` is passed to the container, but for better discoverability for users of your action, we recommended using inputs.
```

Reply:
Hi @lucascosti, sounds great to me! Thanks for the review!